pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -363,11 +401,33 @@ ssize_t sock_dtls_send(sock_dtls_t *sock, sock_dtls_session_t *remote,
         }
     }
 
-    return dtls_write(sock->dtls_ctx, &remote->dtls_session,
-                      (uint8_t *)data, len);
+    res = dtls_write(sock->dtls_ctx, &remote->dtls_session,
+                     (uint8_t *)data, len);
+#ifdef SOCK_HAS_ASYNC
+    if ((res >= 0) && (sock->async_cb != NULL)) {
+        sock->async_cb(sock, SOCK_ASYNC_MSG_SENT, sock->async_cb_arg);
+    }
+#endif /* SOCK_HAS_ASYNC */
+    return res;
 }
 
-static ssize_t _copy_buffer(sock_dtls_t *sock, void *data, size_t max_len)
+#if SOCK_HAS_ASYNC
+static void _check_more_chunks(sock_udp_t *udp_sock, void **data,
```

Comment:
A short description of what this function does / is supposed to do when chunked payloads are implemented would be good.